README.md

```diff
@@ -35,8 +34,12 @@ This section deals with setting up either Windows Subsystem for Linux (WSL)
 or VMWare. 
 If you are using MacOS or Linux you can skip this section.
 
+Follow [this guide](guides/Windows_Subsystem_for_Linux_Installation_Guide_for_Windows_10.md) if you are running Windows 10.
```

Comment:
I would transform these into a sections called WSL and VMWare respectively and in the opening paragraph before would say something like:
If you are using windows 10 we suggest using WSL, if you are using an older windows version we also support running a virtual linux machine with WMWare.

Reply:
Yep, that sounds good, I'll do that